onnx/test/test_with_ort.py

```diff
@@ -2,6 +2,7 @@
 # This file is for testing ONNX with ONNXRuntime during ONNX Release
 # Create a general scenario to use ONNXRuntime with ONNX
 
+
```

Comment:
Looks like no change in this file? Can we drop this redundant blank line change?

Reply:
I added this line because of the flake8 error with tools/style.sh, but I agree it could/should maybe a separate PR (like mentioned here https://github.com/onnx/onnx/issues/4328)